acceleration:
  e0:
    x: -0.002434155094814403
    y: 0.16199778031955867
    z: 0.34992142951989164
  e1:
    x: 0.007484333041571922
    y: 4.219501373370349e-05
    z: 0.025103581495399485
angular_velocity:
  e0:
    x: -0.07729267937060173
    y: -0.04611417110777259
    z: 0.012509592393452989